components/about-page/AboutPage.style.js

```diff
@@ -0,0 +1,31 @@
+import styled from 'styled-components';
+import { min } from '../../lib/responsive';
+
+const HeaderandSquiggleContainer = styled.div`
+  margin-left: -1.5em;
+  @media ${min.tablet} {
+    padding-bottom: 1.5em;
+  }
+  @media ${min.desktop} {
+    padding-bottom: 2em;
+  }
+`;
+
+const SquiggleUnderline = styled.img`
+  position: absolute;
+  width: clamp(9em, 44vw, 10em); //21vw; //clamp(10em, 95vw, 13em);
```

Comment:
is this intended to be part of the stuff you wanted commented out?

Reply:
lol totally didn't mean to commit that